.github/workflows/dependacy_merge.yml

```diff
@@ -0,0 +1,81 @@
+name: Dependabot fetch metadata
+on: pull_request
+
+permissions:
+  pull-requests: write
+  issues: write
+  repository-projects: write
```

Comment:
Why write permission is needed for GITHUB_TOKEN ?

Reply:
Also, PR and Issue writing is being taken care by CHERRYPICK token which already has these permissions so we need not to allow GITHUB_TOKEN these permissions !